static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -99,6 +102,28 @@ function FlamegraphZoomView({
     );
   }, [flamegraphOverlayCanvasRef, flamegraph, flamegraphTheme]);
 
+  const internalSampleTickRenderer: InternalSampleTickRenderer | null = useMemo(() => {
+    if (!isInternalFlamegraphDebugModeEnabled) {
+      return null;
+    }
+
+    if (!flamegraphOverlayCanvasRef || !flamegraphView?.configSpace) {
+      return null;
+    }
+    return new InternalSampleTickRenderer(
+      flamegraphOverlayCanvasRef,
+      flamegraph,
+      flamegraphView.configSpace,
+      flamegraphTheme
+    );
+  }, [
+    isInternalFlamegraphDebugModeEnabled,
+    flamegraphOverlayCanvasRef,
+    flamegraph,
+    flamegraphView?.configSpace,
```

Comment:
You can drop this dep if you dont store configSpace

Reply:
I don't see the benefit int this other than having to now eslint-ignore 🤔 

i can pass it to draw, but we had discussed that we compute this AOT so that's why we've moved to the constructor